src/commands/revision/changeRevisionActiveState.ts

```diff
@@ -3,17 +3,26 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ContainerAppsAPIClient } from "@azure/arm-appcontainers";
-import { IActionContext, nonNullProp } from "@microsoft/vscode-azext-utils";
+import { ContainerAppsAPIClient, KnownActiveRevisionsMode } from "@azure/arm-appcontainers";
+import { IActionContext, nonNullProp, nonNullValue } from "@microsoft/vscode-azext-utils";
 import { ext } from "../../extensionVariables";
 import type { ContainerAppItem } from "../../tree/ContainerAppItem";
 import { RevisionItem } from "../../tree/revisionManagement/RevisionItem";
 import { createContainerAppsClient } from "../../utils/azureClients";
 import { localize } from "../../utils/localize";
 import { pickContainerApp } from "../../utils/pickItem/pickContainerApp";
+import { pickRevision } from "../../utils/pickItem/pickRevision";
 
 export async function executeRevisionOperation(context: IActionContext, node: ContainerAppItem | RevisionItem | undefined, operation: RevisionOperation): Promise<void> {
-    const item = node ?? await pickContainerApp(context);
+    if (!node) {
```

Comment:
What if the `node` is a `ContainerAppItem` but it's in multiple revision mode?

Reply:
I think we should be good with this scenario because the command is removed from the context menu when in multiple revisions mode.  When in multiple revisions mode the command should only be executable from either no node (command-palette) or a `RevisionItem`!